tests/Conversation/CommentForm/CommentForm.js

```diff
@@ -76,6 +76,7 @@ describe('Comment Form', () => {
   });
 
   it('renders CommentFormActions (with correct props)', () => {
```

Comment:
There is a number of new props that have been added e.g. `onBlur` & `onFocus`. Should we add tests for what these do? 

I believe they set state so maybe it's a good idea?